app/src/components/AuthorDetails/AuthorDetails.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+
+export default class AuthorDetails extends Component {
+  constructor( props ) {
+    super( props )
+    this.state = {
+      id: this.props.id,
+      author: {},
+      error: null
+    }
+  }
+
+  componentDidMount() {
+    let fetchString = `/authors/${this.state.id}`
```

Comment:
fetchString seems a little too ambiguous too me. What is this fetch call actually retrieving?